frontend/src/components/user/UserApps.tsx

```diff
@@ -13,13 +14,20 @@ interface Props {
 const UserApps: FunctionComponent<Props> = ({ variant }) => {
   const { t } = useTranslation()
   const user = useUserContext()
+  const userDispatch = useUserDispatch()
 
   const getApps = useCallback(
     () => getAppsInfo(user.info[`${variant}-flatpaks`]),
     [user.info, variant],
   )
   const { execute, status, value: apps } = useAsync(getApps, false)
 
+  const doRefreshDev = useCallback(async () => {
+    await refreshDevFlatpaks()
+    await getUserData(userDispatch)
```

Comment:
I don't really like, that we basically login the user again due to this. I suspect, this is why it seems like the whole page reloads as presumably we're logged out/loading login for a few sec.

It would be much nicer to just replace the `apps` list, but that might be too much to ask? :)